commit/merkleroot/rmn/common.go

```diff
@@ -0,0 +1,44 @@
+package rmn
+
+import (
+	"fmt"
+
+	mapset "github.com/deckarep/golang-set/v2"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-ccip/commit/merkleroot/rmn/rmnpb"
+)
+
+func (c *PBClient) marshalAndSend(req *rmnpb.Request, nodeID uint32) error {
+	reqBytes, err := proto.Marshal(req)
+	if err != nil {
+		return fmt.Errorf("proto marshal: %w", err)
```

Comment:
We use protobuf in many parts of the stack, would be useful to provide more context:

```suggestion
		return fmt.Errorf("proto marshal RMN request: %w", err)
```